scripts/linkchecker.py

```diff
@@ -410,10 +411,33 @@ def validate_links(page):
 
     matches = regex.findall(content)
     records = []
+    target_records = []
     for m in matches:
         r = check_target(page, m[0], m[1])
         if r:
             records.append(r)
+            target_records.append(m[1])
+
+    # if multiple records are the same they need not be checked repeatedly
+    # remove paths that are not relative too
+    target_records = set([item for item in target_records
+                         if not item.startswith("http") and
+                          not item.startswith(f"/{LANG}")])
+
+    if in_place_edit and len(target_records) > 0:
```

Comment:
```suggestion
    if in_place_edit and target_records:
```
Nit: the truthiness value of a set (or any other collection) is already `True` if its length is greater than zero, `False` otherwise.